pages/services/kafka/2.6.0-2.2.1/started/index.md

```diff
@@ -0,0 +1,617 @@
+
+*   **In DC/OS CLI options.json**: `placement-strategy`: `ANY` or `NODE` (default: `ANY`)
+*   **DC/OS web interface**: `PLACEMENT_STRATEGY`: `ANY` or `NODE`
+
+## Configure Kafka Broker Properties
+
+Kafka Brokers are configured through settings in a server.properties file deployed with each Broker. The settings here can be specified at installation time or during a post-deployment configuration update. They are set in the DC/OS Universe's config.json as options such as:
+
+```json
+    "log_retention_hours": {
+        "title": "log.retention.hours",
+        "description": "Override log.retention.hours: The number of hours to keep a log file before deleting it (in hours), tertiary to log.retention.ms property",
+        "type": "integer",
+        "default": 168
+    },
+```
+
+The defaults can be overridden at install time by specifying an options.json file with a format like this:
+
+```json
+    {
+        "kafka": {
+            "log_retention_hours": 100
+        }
+    }
+```
+
+These same values are also represented as environment variables for the scheduler in the form `KAFKA_OVERRIDE_LOG_RETENTION_HOURS` and may be modified through the DC/OS web interface and deployed during a rolling upgrade as [described here][12].
+
+<a name="disk-type"></a>
+## Disk Type
+
+The type of disks that can be used for storing broker data are: `ROOT` (default) and `MOUNT`.  The type of disk may only be specified at install time.
+
+* `ROOT`: Broker data is stored on the same volume as the agent work directory. Broker tasks will use the configured amount of disk space.
+* `MOUNT`: Broker data will be stored on a dedicated volume attached to the agent. Dedicated MOUNT volumes have performance advantages and a disk error on these MOUNT volumes will be correctly reported to Kafka.
+
+Configure Kafka service to use dedicated disk volumes:
+* **DC/OS cli options.json**:
+
+```json
+    {
+        "brokers": {
+            "disk_type": "MOUNT"
+        }
+    }
+```
+
+* **DC/OS web interface**: Set the environment variable `DISK_TYPE`: `MOUNT`
+
+When configured to `MOUNT` disk type, the scheduler selects a disk on an agent whose capacity is equal to or greater than the configured `disk` value.
+
+## JVM Heap Size
+
+Kafka service allows configuration of JVM Heap Size for the broker JVM process. Use the following configuration options:
+
+```json
+    {
+        "brokers": {
+            "heap": {
+                "size": 2000
+            }
+        }
+    }
+```
+
+* **DC/OS web interface**: Set the environment variable `BROKER_HEAP_MB`: `2000`
+
+**Note**: The total memory allocated for the Mesos task is specified by the `BROKER_MEM` configuration parameter. The value for `BROKER_HEAP_MB` should not be greater than `BROKER_MEM` value. Also, if `BROKER_MEM` is greater than `BROKER_HEAP_MB` then the Linux operating system will use `BROKER_MEM` - `BROKER_HEAP_MB` for [PageCache](https://en.wikipedia.org/wiki/Page_cache).
```

Comment:
Use the `Note` message format represented 
 in https://wiki.mesosphere.com/display/DOCS/Documentation+Style+Guide#DocumentationStyleGuide-Notes